docs/src/introduction_to_catalyst/introduction_to_catalyst.md

```diff
@@ -81,14 +81,14 @@ latexify(rn)
 ```@example tut1
 rn #hide
 ```
-Assuming [Graphviz](https://graphviz.org/) is installed and command line
-accessible, within a Jupyter notebook we can also graph the reaction network by
-```julia
-g = Graph(rn)
+Catalyst also has functionality for visualizing networks using the [Makie](https://docs.makie.org/stable/)
+plotting ecosystem. The relevant packages to load are Catalyst, GraphMakie, and a Makie backend
+such as GLMakie. Doing so and then using the `plot_network` function allows us to 
+visualize the network: 
+```@example tut1
+using Catalyst, GraphMakie, GLMakie
```

Comment:
Can you just import `GraphMakie` and `GLMakie` and still use `plot_network`? If so, then you could still `using Plots` which is probably a more relevant workflow for most users.